src/main/java/org/mifos/processor/bulk/api/CallbackController.java

```diff
@@ -0,0 +1,53 @@
+package org.mifos.processor.bulk.api;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import org.mifos.processor.bulk.schema.AuthorizationResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.mifos.processor.bulk.zeebe.ZeebeVariables.*;
+
+@RestController
+public class CallbackController {
+
+    @Autowired
+    private ZeebeClient zeebeClient;
+
+    protected Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private static final String EXPECTED_AUTH_STATUS = "Y";
+
+
+    @PostMapping("/authorization/callback")
```

Comment:
can you build this API as per the REST API standards

Reply:
check the URL naming convention

Reply:
What should be the endpoint? or Where is this documentation?

Reply:
/batchtransactions?command=authorised

Reply:
Mentioned in backlog ticket **https://fynarfin.atlassian.net/browse/PHEE-449**